interview-questions.md

```diff
@@ -7,53 +7,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
+  Your answer: While I know MVC stands for model-view-controller, I cant recall what that exactly means. I will do some research on that and get back to you.
 
-  Researched answer:
+  Researched answer: Model-view-controller is a framework that provides default structures for a database, a web service, and web pages. 
```

Comment:
This is a good answer but in an interview you may want to elaborate on this a bit more. What does each of those things do? how to they relate to M, C, and V.